internal/datacoord/compaction_trigger_v2.go

```diff
@@ -260,6 +296,72 @@ func (m *CompactionTriggerManager) SubmitL0ViewToScheduler(ctx context.Context,
 		return
 	}
 
+	// add l0 import task for the collection if the collection is importing
+	importJobs := m.imeta.GetJobBy(ctx, WithCollectionID(collection.ID), WithoutJobStates(internalpb.ImportJobState_Completed, internalpb.ImportJobState_Failed))
```

Comment:
Wrap an independent method to add the l0 import task.

Reply:
done